docs/penalty-box.raml

```diff
@@ -0,0 +1,56 @@
+#%RAML 0.8
+---
+title: Penalty Box
+baseUri: "http://penalty-box.yourdomain.com"
+documentation:
+  - title: "Penalty Box: A simple restful rate limiter"
+    content: |
+      Penalty box exposes an API that lets multiple service use it as a restful rate limiter. That API is documented here.
+
+      You may also be interested in [the GitHub repository](http://github.com/gamechanger/lox)
+/health:
+  description: Health check endpoint
+  get:
+    description: Returns 200 if the service is available
+    responses:
+      200:
+        description: The Penalty Box service is available and serving requests
+/rate-limit:
+  description: Checks whether a request is allowed and keeps track of how often a key has been accessed
+  post:
+    description: Post info on a request to see if request is allowed. If allowed Penalty Box appends keep track of it to enforce rate limits
+    queryParameters:
+      appName:
```

Comment:
sort of arbitrary based off of what travis did. but looking for the 'right' answer right now and will change it whatever i find.